ee/query-service/app/api/license.go

```diff
@@ -102,7 +102,7 @@ func (ah *APIHandler) applyLicense(w http.ResponseWriter, r *http.Request) {
 		RespondError(w, model.BadRequest(fmt.Errorf("license key is required")), nil)
 		return
 	}
-	license, apiError := ah.LM().Activate(r.Context(), l.Key)
```

Comment:
remove the `UseLicensesV3` flag from line 87 as well ?

Reply:
Yes missed that